pipenv/utils/resolver.py

```diff
@@ -518,7 +518,7 @@ def pip_options(self):
             pip_options, _ = self.pip_command.parser.parse_args(self.pip_args)
             pip_options.cache_dir = self.project.s.PIPENV_CACHE_DIR
             pip_options.no_python_version_warning = True
-            pip_options.no_input = True
+            pip_options.no_input = self.project.settings.get("disable_pip_input", True)
```

Comment:
I think this is going to at a minimum require some documentation around how to specify this in the Pipfile and what it does.

Reply:
Sure -- so you have to go into the `docs` directory and there is a separate `requirements.txt` in there to install using `pip -r requirements.txt`.  I think after that you should just have to run `make html` to build the html docs.  Going from memory here but I did it not so long ago ...

Reply:
I can't get the pipfile to show as syntax. But the doc is added. Maybe you can take it from here, @matteius ?

Reply:
since its in your fork, best I can do is address any formatting in a followup PR.  I'll at least get this merged now.